packages/react-core/src/hooks/__tests__/useGetUrl.test.ts

```diff
@@ -0,0 +1,195 @@
+import { renderHook } from '@testing-library/react-hooks';
+import { waitFor } from '@testing-library/react';
+import { getUrl } from 'aws-amplify/storage';
+
+import { useGetUrl, UseGetUrlInput } from '../useGetUrl';
+
+jest.mock('aws-amplify/storage');
+
+describe('useGetUrl', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  const url = 'https://amplify.s3.amazonaws.com/path/to/the/file.jpg';
+
+  const onStorageGetError = jest.fn();
+  const onGetUrlError = jest.fn();
+
+  const KEY_INPUT: UseGetUrlInput = {
+    key: 'file.jpg',
+    options: { accessLevel: 'guest' },
+    onError: onStorageGetError,
+  };
+
+  const PATH_INPUT: UseGetUrlInput = {
+    path: 'guest/file.jpg',
+    onError: onGetUrlError,
+  };
+
+  const paramType = [
```

Comment:
`paramType` as well as the above test variables should live outside the `describe` block